examples/braket_features/Using_rigetti_calibration_data_in_noise_channel_parameters/Using_rigetti_calibration_data_in_noise_channel_parameters.ipynb

```diff
@@ -0,0 +1,1151 @@
+{
```

Comment:
<p>Typically for AWS style, you'd want an action verb, rather than a gerund, like "Use Rigetti..." and you probably don't need "- An example" since this repo is all about examples.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>